openstudiocore/src/model/EnergyManagementSystemActuator.cpp

```diff
@@ -116,11 +149,74 @@ namespace detail {
     return result;
   }
 
+  void EnergyManagementSystemActuator_Impl::resetZoneName() {
+    bool result = setString(OS_EnergyManagementSystem_ActuatorFields::ZoneName, "");
+    OS_ASSERT(result);
+  }
+
 } // detail
 
 EnergyManagementSystemActuator::EnergyManagementSystemActuator(const ModelObject& modelObject, const std::string actuatedComponentType, const std::string actuatedComponentControlType)
   : ModelObject(EnergyManagementSystemActuator::iddObjectType(), modelObject.model()) {
   OS_ASSERT(getImpl<detail::EnergyManagementSystemActuator_Impl>());
+  //Do not use this constructor for spaceloads that are defined in SpaceType objects
+  bool ok = setActuatedComponent(modelObject);
+  if (!ok) {
+    remove();
```

Comment:
This is a pattern (remove in constructor) that I have not seen or used. If it works it seems like a pretty good thing to do. Have you @brianlball verified that it does? Given that you are doing this in the wrapper class I think it probably does work.